include/stl2/view/subrange.hpp

```diff
@@ -23,6 +23,16 @@
 #include <stl2/view/view_interface.hpp>
 
 STL2_OPEN_NAMESPACE {
+	template<class From, class To>
+	STL2_CONCEPT _ConvertibleToNotSlicing =
+		ConvertibleTo<From, To> &&
+		// A conversion is a slicing conversion if the source and the destination
+		// are both pointers, and if the pointed-to types differ after removing
+		// cv qualifiers.
+		!(_Is<__uncvref<From>, std::is_pointer> &&
+		  _Is<__uncvref<To>, std::is_pointer> &&
+		  _NotSameAs<std::remove_pointer_t<__uncvref<From>>, std::remove_pointer_t<__uncvref<To>>>);
```

Comment:
The comment says "after removing cv qualifiers":
```suggestion
		  _NotSameAs<remove_cv_t<std::remove_pointer_t<__uncvref<From>>>, remove_cv_t<std::remove_pointer_t<__uncvref<To>>>>);
```

Reply:
Although we usually defer to the core language rules for conversion of pointers-to-array. e.g. for `shared_ptr<T[]>`'s constructors. So maybe:
```suggestion
		  std::is_convertible_v<std::remove_pointer_t<__uncvref<From>>(*)[],
		                        std::remove_pointer_t<__uncvref<To>>(*)[]>);
```